server/BUILD

```diff
@@ -61,11 +72,15 @@ native_java_libraries(
 
         # External dependencies from Maven
         "@maven//:com_google_code_findbugs_jsr305",
+        "@maven//:com_google_guava_guava",
```

Comment:
hmmm we should not be importing guava unless we _really_ have to! It's a large dependency we've moved away from. When we need something from Guava we either re-implement it or find another way right now

Reply:
unfortunately this is needed because we need it to compile grpc.